ssz/src/main/java/org/ethereum/beacon/ssz/SSZSerializerBuilder.java

```diff
@@ -50,6 +50,16 @@ public SSZSerializerBuilder(SSZSchemeBuilder schemeBuilder, SSZCodecResolver cod
     this.sszSerializer = new SSZSerializer(schemeBuilder, codecResolver, sszModelFactory);
   }
 
+  public SSZSerializerBuilder initWith(SSZSerializer sszSerializer, SSZCodecResolver codecResolver) {
```

Comment:
Maybe keep builder simple and conventional? I mean use plain `withSszSerializer`, `withCodecResolver`, etc. methods?

Reply:
rebuilt